venv_update.py

```diff
@@ -205,7 +205,10 @@ def exec_scratch_virtualenv(args):
         rename(tmp, scratch.src)
 
     import sys
-    if sys.prefix != scratch.venv:
+    from os.path import abspath
```

Comment:
I haven't seen this pattern of importing on the fly in other Yelp code, but I figured it was good to keep consistency.  If we'd like to move away from this and import everything at the top, just let me know and I'm happy to make that happen.